typescript/web/src/components/website/Navbar/Submenu.tsx

```diff
@@ -30,8 +30,8 @@ const DesktopSubmenu = (props: SubmenuProps) => {
       </NavLink.Desktop>
 
       <NavMenu {...getMenuProps()} animate={isOpen ? "open" : "closed"}>
-        <Box maxW="7xl" mx="auto" px="8">
-          <SimpleGrid spacing="10" columns={2}>
+        <Box maxW="7xl" mx="auto" px="400">
+          <SimpleGrid spacing="" columns={2}>
```

Comment:
If spacing isn't needed, you can just remove the property:

```suggestion
          <SimpleGrid columns={2}>
```